src/panels/OriginPanel/OriginPanelMenuButton.tsx

```diff
@@ -34,14 +31,16 @@ export function OriginPanelMenuButton({ id }: Props) {
     );
   }
 
-  return hasDistinctAim ? (
-    <AnchorAimButtons
-      anchorName={anchorNode?.name}
-      aimName={aimNode?.name}
-      isOpenSpaceReady={isReady}
-      id={id}
-    />
-  ) : (
-    <FocusButton anchorName={anchorNode?.name} isOpenSpaceReady={isReady} id={id} />
+  if (!isReady) {
+    return (
+      <TaskBarMenuButton id={id} disabled>
+        <LoadingBlocks n={1} w={84} />
+      </TaskBarMenuButton>
+    );
```

Comment:
While I like the simplicty of this, I think we might want the focus icon to still be there?  At least if, as we've discussed, the longer-term plan is to not disable the menu button while loading. Then, the user needs a way to know which menu this is. 

To clarify, I suggest to make this button look the way it did before
![image](https://github.com/user-attachments/assets/f23fdb38-f42d-4a57-b605-8a36deadb7d0)
at least including the icon, but personally I would prefer to add the focus part as well, or a loading block that represents that text.  The blocks makes the most sense if they visually represent the thing that is loading directly, at least IMO

Reply:
Technically we don't know if this will show the anchor / aim or just the focus button 🤔 I guess we could always make the placeholder the focus anyway 🤷 but if we want to remove this functionality altogether it doesn't make sense to change it imo

Reply:
Yeah making the placeholder the focus makes sense - or at least give it the focus icon since that's the once we use for navigation. 

Also, we wouldn't know if we're in anchor/aim mode, and the general assumption can be that we probably aren't since Focus is default :)